app/src/debug/res/raw/cert.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
why do we have a cert in github ?

Reply:
It's a public key to be added as a trusted certifier for the debug versions of the app. This allows https connection using the keys signed with private part of this key. This method is only used on degug, the release uses standard key validation